src/som/primitives/actors/PromisePrims.java

```diff
@@ -51,23 +60,46 @@ public boolean matches(final Object[] args, final ExpressionNode[] argNodes) {
              specializer = IsActorModule.class, noWrapper = true)
   public abstract static class CreatePromisePairPrim extends UnaryExpressionNode {
 
+    protected final AbstractBreakpointNode promiseResolverBreakpoint;
```

Comment:
This can not work.
Truffle nodes need to be marked with the `@Child` annotation, like in the other places.
Also, the field cannot be marked as final, because the node needs to be able to change itself to a specialized version.